dist/.magento.env.yaml

```diff
@@ -58,6 +58,17 @@
 #            deploy:                                                                                                  #
 #              SCD_STRATEGY: "compact"                                                                                #
 #######################################################################################################################
+# SCD_MAX_EXECUTION_TIME - allows you to increase maximum expected execution time of static content deploy command.   #
+#                          By default in Magento this value is set to 400 seconds but in some scenarios               #
```

Comment:
By default, the Magento Commerce application sets the maximum expected execution to 400 seconds, but in some scenarios you might need more time to complete the static content deployment .

Reply:
Fixed, thank you.